src/Core/View.js

```diff
@@ -32,6 +32,7 @@ export const VIEW_EVENTS = {
     INITIALIZED: 'initialized',
     COLOR_LAYERS_ORDER_CHANGED,
     CAMERA_MOVED: 'camera-moved',
+    REMOVED: 'removed',
```

Comment:
Can you rename to `DISPOSED` please ? It's more commonly used in threejs and it will be the same name than the method